docker/startup

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+RCLONE_CONFIG=/config/rclone/rclone.conf
+
+if [ -z "$REMOTE_NAME" ]; then
+    echo "REMOTE_NAME is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_URL" ]; then
+    echo "REMOTE_URL is not set"
+    exit 1
+fi
+if [ -z "$REMOTE_VENDOR" ]; then
+    echo "REMOTE_VENDOR is not set"
+    exit 1
+fi
+
+mkdir -p "$(dirname "$RCLONE_CONFIG")"
+
+cat <<EOF >"$RCLONE_CONFIG"
+[$REMOTE_NAME]
+type = webdav
+url = $REMOTE_URL
+vendor = $REMOTE_VENDOR
+nextcloud_chunk_size = 0
+EOF
+
+# parse args
+ifsBak=$IFS
+IFS=' '
+
+ARGS="$REMOTE_NAME:"
+for arg in $PROXY_ARGS; do
+    ARGS="$ARGS $arg"
+done
+
+IFS=$ifsBak
+
+rclone serve s3 "$ARGS"
```

Comment:
when running `s3-proxy` with `-e PROXY_ARGS="--vfs-cache-mode=off"` it gives error as : 

```
WARNING: Published ports are discarded when using host network mode
2023/06/30 06:00:09 NOTICE: Chunks temporary upload directory: http://localhost/nextcloud-master/remote.php/webdav/
2023/06/30 06:00:09 Failed to create file system for "nc: --vfs-cache-mode=off": read metadata failed: No 'Authorization: Basic' header found. Either the client didn't send one, or the server is misconfigured, No 'Authorization: Bearer' header found. Either the client didn't send one, or the server is mis-configured: Sabre\DAV\Exception\NotAuthenticated: 401 Unauthorized
```

Reply:
Now i can run it with it.